ports/stm32/boards/STM32H747I_DISCO/mpconfigboard.h

```diff
@@ -0,0 +1,326 @@
+#define MICROPY_HW_FMC_A3           (pin_F3)
+#define MICROPY_HW_FMC_A4           (pin_F4)
+#define MICROPY_HW_FMC_A5           (pin_F5)
+#define MICROPY_HW_FMC_A6           (pin_F12)
+#define MICROPY_HW_FMC_A7           (pin_F13)
+#define MICROPY_HW_FMC_A8           (pin_F14)
+#define MICROPY_HW_FMC_A9           (pin_F15)
+#define MICROPY_HW_FMC_A10          (pin_G0)
+#define MICROPY_HW_FMC_A11          (pin_G1)
+#define MICROPY_HW_FMC_A12          (pin_G2)
+#define MICROPY_HW_FMC_A14          (pin_G4)
+#define MICROPY_HW_FMC_A15          (pin_G5)
+#define MICROPY_HW_FMC_D0           (pin_D14)
+#define MICROPY_HW_FMC_D1           (pin_D15)
+#define MICROPY_HW_FMC_D2           (pin_D0)
+#define MICROPY_HW_FMC_D3           (pin_D1)
+#define MICROPY_HW_FMC_D4           (pin_E7)
+#define MICROPY_HW_FMC_D5           (pin_E8)
+#define MICROPY_HW_FMC_D6           (pin_E9)
+#define MICROPY_HW_FMC_D7           (pin_E10)
+#define MICROPY_HW_FMC_D8           (pin_E11)
+#define MICROPY_HW_FMC_D9           (pin_E12)
+#define MICROPY_HW_FMC_D10          (pin_E13)
+#define MICROPY_HW_FMC_D11          (pin_E14)
+#define MICROPY_HW_FMC_D12          (pin_E15)
+#define MICROPY_HW_FMC_D13          (pin_D8)
+#define MICROPY_HW_FMC_D14          (pin_D9)
+#define MICROPY_HW_FMC_D15          (pin_D10)
+#define MICROPY_HW_FMC_D16          (pin_H8)
+#define MICROPY_HW_FMC_D17          (pin_H9)
+#define MICROPY_HW_FMC_D18          (pin_H10)
+#define MICROPY_HW_FMC_D19          (pin_H11)
+#define MICROPY_HW_FMC_D20          (pin_H12)
+#define MICROPY_HW_FMC_D21          (pin_H13)
+#define MICROPY_HW_FMC_D22          (pin_H14)
+#define MICROPY_HW_FMC_D23          (pin_H15)
+#define MICROPY_HW_FMC_D24          (pin_I0)
+#define MICROPY_HW_FMC_D25          (pin_I1)
+#define MICROPY_HW_FMC_D26          (pin_I2)
+#define MICROPY_HW_FMC_D27          (pin_I3)
+#define MICROPY_HW_FMC_D28          (pin_I6)
+#define MICROPY_HW_FMC_D29          (pin_I7)
+#define MICROPY_HW_FMC_D30          (pin_I9)
+#define MICROPY_HW_FMC_D31          (pin_I10)
+
+#define MICROPY_HW_USB_VID                      0x0483
+#define MICROPY_HW_USB_PID                      0x5740
+#define MICROPY_HW_USB_PID_CDC_MSC              (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC_HID              (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC                  (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_MSC                  (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC2_MSC             (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC2                 (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC3                 (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC3_MSC             (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC_MSC_HID          (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC2_MSC_HID         (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_PID_CDC3_MSC_HID         (MICROPY_HW_USB_PID)
+#define MICROPY_HW_USB_LANGID_STRING            0x409
```

Comment:
This is not needed, it's the default.

I think this was copied from the Portenta H7.  I think just remove all this custom USB config.